experiment/coverage_utils.py

```diff
@@ -49,40 +48,42 @@ def upload_coverage_info_to_bucket():
 
 def generate_coverage_reports(experiment_config: dict):
     """Generates coverage reports for each benchmark and fuzzer."""
-    logger.info('Start generating coverage report for benchmarks.')
+    logger.info('Start generating coverage reports.')
+
     benchmarks = experiment_config['benchmarks'].split(',')
     fuzzers = experiment_config['fuzzers'].split(',')
     experiment = experiment_config['experiment']
-    with multiprocessing.Pool() as pool:
-        generate_coverage_report_args = [(experiment, benchmark, fuzzer)
-                                         for benchmark in benchmarks
-                                         for fuzzer in fuzzers]
-        pool.starmap(generate_coverage_report, generate_coverage_report_args)
-    logger.info('Finished generating coverage report.')
+
+    for benchmark in benchmarks:
+        for fuzzer in fuzzers:
+            generate_coverage_report(experiment, benchmark, fuzzer)
+
+    logger.info('Finished generating coverage reports.')
 
 
 def generate_coverage_report(experiment, benchmark, fuzzer):
     """Generates the coverage report for one pair of benchmark and fuzzer."""
     logs.initialize()
-    logger.info('Generating coverage report for benchmark: {benchmark} \
-                fuzzer: {fuzzer}.'.format(benchmark=benchmark, fuzzer=fuzzer))
-    generator = CoverageReporter(fuzzer, benchmark, experiment)
+    logger.info(
+        ('Generating coverage report for '
+         'benchmark: {benchmark} fuzzer: {fuzzer}.').format(benchmark=benchmark,
+                                                            fuzzer=fuzzer))
+
+    generator = CoverageReporter(experiment, fuzzer, benchmark)
```

Comment:
should probably be called `coverage_reporter`.

Reply:
Done!